KomaMRICore/Project.toml

```diff
@@ -7,27 +7,19 @@ version = "0.7.7"
 Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
 CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
 Functors = "d9f16b24-f501-4c13-a1f2-28368ffc5196"
-Interpolations = "a98d9a8b-a2ab-59e6-89dd-64a1c18fca59"
-MAT = "23992714-dd62-5051-b70f-ba57cb901cac"
-MRIBase = "f7771a9a-6e57-4e71-863b-6e4b6a2f17df"
-Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
+KomaMRIBase = "d0bc0b20-b151-4d03-b2a4-6ca51751cb9c"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
-Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 Scanf = "6ef1bc8b-493b-44e1-8d40-549aa65c4b41"
```

Comment:
I dont think this is being used in Core.

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/80d9eef154cb4f94280fb6cdb3782a180a9f3fbb